lib/walk.js

```diff
@@ -28,6 +28,11 @@ const generateRecursiveFilePromise = async (dir, file) => {
   });
 };
 
+/**
+ * @deprecated
+ * Use the corresponding export from local-dev-lib
+ * https://github.com/HubSpot/hubspot-local-dev-lib
+ */
```

Comment:
Might be helpful to add the new import paths for those that have changed. In this case its `@hubspot/local-dev-lib/fs`

Reply:
Good idea. Added in [a821417](https://github.com/HubSpot/cli-lib/pull/39/commits/a821417f90003cb28b571909a63f011d96365049)